docs/controls/BubbleSingleDataset.md

```diff
@@ -0,0 +1,327 @@
+# Bubble Single Dataset
+
+A native bubble graph using D3 based on standard design patterns.
+
+-   [Bubble](#bubble)
+    -   [Usage](#usage)
+        -   [Structure](#structure)
+            -   [Weight Based Bubble](#weight-based-bubble)
+            -   [Color Based Bubble](#color-based-bubble)
+            -   [Weight Color Based Bubble](#weight-color-based-bubble)
+            -   [Custom Size Based Bubble](#custom-size-based-bubble)
+    -   [JSON Properties](#json-properties)
+        -   [Root](#root)
+        -   [Data](#data)
+            -   [Required](#required)
+            -   [Optional](#optional)
+        -   [Values](#values)
+            -   [Required](#required-2)
+            -   [Optional](#optional-2)
+        -   [Regions](#regions)
+            -   [Required](#required-3)
+            -   [Optional](#optional-3)
+        -   [Constraints](#constraints)
+
+## Usage
+
+### Structure
+
+You will **not** need all the properties in the example below.
+Check out _optional_/_required_ properties explained in the [JSON Properties](#json-properties) section.
+
+```javascript
+var root = {
+    bindTo: "#root",
+    axis: {
+        x: {
+            type: Carbon.helpers.AXIS_TYPE.TIME_SERIES,
+            label: "Some X Label",
+            lowerLimit: "2016-01-01T12:00:00Z",
+            upperLimit: "2017-01-01T12:00:00Z"
+        },
+        y: {
+            label: "Some Y Label",
+            lowerLimit: 0,
+            upperLimit: 20
+        }
+    }
+};
+
+var data = {
+    key: "uid_1",
+    label: {
+        display: "Data Label 1"
+    },
+    color: Carbon.helpers.COLORS.BLUE,
+    values: [
+        {
+            x: "2016-02-03T12:00:00Z",
+            y: 4
+        },
+        {
+            x: "2016-05-01T12:00:00Z",
+            y: 15
+        },
+        {
+            x: "2016-10-01T12:00:00Z",
+            y: 10
+        }
+    ]
+};
+var bubbleDefault = Carbon.api.graph(root);
+bubbleDefault.loadContent(Carbon.api.bubble(data));
```

Comment:
Will we use Carbon.api.bubble in single data set?

Reply:
This was an accidental commit, i already fixed it. 
https://github.com/cerner/carbon-graphs/pull/308/commits/094103a7a84cee171af87eab44dd139833600078